source/core/Variants.cpp

```diff
@@ -162,14 +171,13 @@ VIREO_FUNCTION_SIGNATURET(GetVariantAttribute, GetVariantAttributeParamBlock)
             auto attributeMapIter = attributeMap->find(name);
             if (attributeMapIter != attributeMap->end()) {
                 TypeRef foundValue = attributeMapIter->second;
+                found = true;
```

Comment:
Moved this out here because the found terminal should return true when it finds an attribute, regardless of whether it matches the default data's type.

Reply:
Undoing this change. I will look into it in a separate PR.